index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
```

Comment:
Be weary of you indentation. You always want to push up semantically correct code. Also don't forget to make your site responsive.